cslunetta/musicSpot
# Created Login and registration components

# Login and Registration 
#1 Forms presented to the user to access the application
✔️ New Feature
❌  Bug


## Register
When the use clicks the link ```Not a member yet?``` a form will be displayed to enter in user information to be saved to the database if the email does not exist in the database. When the user completes the form and clicks the ```sign-in``` button they will be able to view the main page. 

## Login
When the user opens the application they are presented with a login form. When the user completes the form and clicks the ```sign-in``` button they will be able to view the main page.

## Testing

1. After opening the application try to login before registering.
2. User should be prompted with a message saying ```User does not exist```
3. now click on the ```Not a member yet?``` link, fill out the form and click the ```sign-in``` button
4. ❌ user should automatically be redirected to the home page of the application.
> Bug - App does not currently connect to home page. Issue ticket has been created to correct this.
5. remove ```localstorage``` key for ```current_user``` from the dev tools to allow application to log out.
6. refresh page to see login screen again.
7. ❌ use the added email address to be able to login.
> Bug - App does not currently connect to home page. Issue ticket has been created to correct this.
